modules/pytket-qiskit/tests/qiskit_convert_test.py

```diff
@@ -126,22 +126,23 @@ def test_convert() -> None:
 def test_symbolic() -> None:
     pi2 = Symbol("pi2")
     pi3 = Symbol("pi3")
-
+    pi0 = Symbol("pi0")
     tkc = Circuit(3, 3, name="test").Ry(pi2, 1).Rx(pi3, 1).CX(1, 0)
+    tkc.add_phase(Symbol("pi0") * 2)
     USquashIBM().apply(tkc)
 
     qc = tk_to_qiskit(tkc)
     tkc2 = qiskit_to_tk(qc)
 
-    assert tkc2.free_symbols() == {pi2, pi3}
-    tkc2.symbol_substitution({pi2: pi / 2, pi3: pi / 3})
+    assert tkc2.free_symbols() == {pi2, pi3, pi0}
+    tkc2.symbol_substitution({pi2: pi / 2, pi3: pi / 3, pi0: 0.1})
 
     backend = Aer.get_backend("statevector_simulator")
     qc = tk_to_qiskit(tkc2)
     assert qc.name == tkc.name
     job = execute([qc], backend)
     state1 = job.result().get_statevector(qc)
-    state0 = np.array(
+    state_phase_ignored = np.array(
```

Comment:
Is it worth keeping this part of the test at all? Given that we go on to make a stronger assertion?

Reply:
No I guess not, it was just useful for me to check I hadn't altered existing behaviour. Will remove.